### PR TITLE
Add test case for first token last token bug

### PR DESCRIPTION
**Pull request by brndysgit**
Funny bug where if a first token is also the last token (index 0) then we will fail to get the next token as it checks for "None". This adds a test case to ensure we took care of that edge case.

### CODE REVIEW

**Review by sindhuvahinis — APPROVED** [review]

File: engines/python/setup/djl_python/tests/test_rolling_batch.py [hunk 1]
```diff
@@ -37,6 +37,48 @@ def test_json_fmt(self):
...
+                "max_new_tokens": 256,
+                "stream": False,
+                "details": True
+            },
+        )
+        req2 = Request(req_input2)
+        for req in [req1, req2]:
+            req.request_output.set_next_token(Token(244, "He", -0.334532),
+                                              is_last_token=True,
+                                              finish_reason="length")
```
[the remark this answers is not in the document]
Comment (brndysgit) [reply engines/python/setup/djl_python/tests/test_rolling_batch.py]: yes, fixed!

File: engines/python/setup/djl_python/tests/test_rolling_batch.py [hunk 2]
```diff
@@ -37,6 +37,53 @@ def test_json_fmt(self):
...
+        request_id=0,
+        input_text="This is a wonderful day",
+        parameters={
+            "max_new_tokens": 256,
+            "details": True
+        },
+        output_formatter=_json_output_formatter,
+    )
+    req1 = Request(req_input1)
+    req_input2 = TextInput(
```
Comment (sindhuvahinis) [inline engines/python/setup/djl_python/tests/test_rolling_batch.py:52]: What's the diff between req_input1 and req_input2? Both are using the same output_formatter.

Comment (sindhuvahinis) [reply engines/python/setup/djl_python/tests/test_rolling_batch.py:52]: There looks like an indentation problem.

Comment (brndysgit) [reply engines/python/setup/djl_python/tests/test_rolling_batch.py:52]: ah sorry, i think i was playing around with Stream and forgot to remove the test. there's no reason to have req 2 if stream isn't making any functional difference.

**Review by sindhuvahinis — APPROVED** [review]
